packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -38,7 +38,7 @@ const firstExistingValue = (...argumentList) => {
  * the built in functions are also considered as Symbol Node (which is similar to variables).
  * So we want to exclude them when getting the variables.
  */
-const BUILT_IN_FUNCTIONS = ['equalText'];
+const BUILT_IN_FUNCTIONS = ['equalText', 'round'];
```

Comment:
This may prove to grow rather annoying over time... @chris-bes I wonder if we can come up with a long term plan to handle needing to do this?